app/static/src/app/store/run/mutations.ts

```diff
@@ -102,6 +103,20 @@ export const mutations: MutationTree<RunState> = {
         delete state.parameterSetResults[parameterSetName];
     },
 
+    [RunMutation.SwapParameterSet](state: RunState, parameterSetName: string) {
+        const paramSet = state.parameterSets.find((set: ParameterSet) => set.name === parameterSetName);
+        if (paramSet && state.parameterValues) {
+            const temp = { ...state.parameterValues };
+            state.parameterValues = paramSet.parameterValues;
+            paramSet.parameterValues = temp;
+        }
+        if (state.parameterSetResults[parameterSetName] && state.resultOde) {
+            const temp = { ...state.resultOde };
```

Comment:
Apologies for this. I had an issue a while back with react doing too many re-renders or updating temp state after I defined it because the object was changing. It seems like I do not need to worry about this in Vue (or maybe typescript). I have removed all the splats.